test/input_files/v4.2/passed/complexfile_passed_000.vcf

```diff
@@ -51,9 +51,9 @@
+1	53234	rs199502715	CAT	]1:1234]CTG	227	PASS	AA=CAT;AC=10;AF=0.0046;AFR_AF=0.02;AMR_AF=0.0028;AN=2184;AVGPOST=0.9936;ERATE=0.0007;LDAF=0.0074;RSQ=0.6237;THETA=0.0119;VT=INDEL	GT:DS:GL	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-0.30,-6.10	0|0:0.100:0,0,0	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-1.20,-24.40	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-1.20,-24.40	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.90,-17.90	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0.00,-0.30,-3.80	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0.00,-0.60,-9.00	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0,0,0	0|0:0.000:0.00,-3.00,-60.80	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-0.90,-18.30	0|0:0.050:0,0,0	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0,0,0	0|0:0.000:0.00,-1.50,-30.50	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-1.20,-24.40	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.050:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-1.20,-24.40	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.050:0,0,0	0|0:0.000:0.00,-0.90,-18.30	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-0.30,-6.10	0|0:0.050:0,0,0	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-1.20,-24.40	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.050:0,0,0	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.60,-12.20	0|0:0.000:0,0,0	0|0:0.000:0.00,-0.30,-6.10	0|0:0.000:0,0,0	0|0:0.050:0,0,0
```

Comment:
in this case I would keep both versions: a SV with one allele, and a multiallelic SV